src/main/java/hudson/plugins/ec2/ssh/EC2UnixLauncher.java

```diff
@@ -314,7 +319,21 @@ private File createIdentityKeyFile(EC2Computer computer) throws IOException {
         EC2PrivateKey ec2PrivateKey = computer.getCloud().resolvePrivateKey();
         String privateKey = "";
         if (ec2PrivateKey != null){
+            LOGGER.log(Level.INFO, () -> "using global private key");
```

Comment:
Not interesting enough to pollute the system log.

```suggestion
            LOGGER.fine(() -> "using global private key for " + computer.getName());
```